src/data/decks.asm

```diff
@@ -163,21 +163,2770 @@ StrangePsyshockDeck: ; 595d2 (16:55d2)
 StarterDeck: ; 59600 (16:5600)
 	dbw 14, FIRE_ENERGY
 	dbw 12, FIGHTING_ENERGY
-	dbw  4, CHARMANDER2
+	dbw  4, CHARMANDER_LV10
 	dbw  3, CHARMELEON
-	dbw  2, GROWLITHE1
-	dbw  3, PONYTA2
-	dbw  2, DIGLETT1
-	dbw  4, MACHOP2
-	dbw  3, MACHOKE3
-	dbw  2, RATTATA1
+	dbw  2, GROWLITHE_LV12
+	dbw  3, PONYTA_LV10
+	dbw  2, DIGLETT_LV8
+	dbw  4, MACHOP_LV20
+	dbw  3, MACHOKE_LV40
+	dbw  2, RATTATA_LV9
 	dbw  1, RATICATE
-	dbw  2, SPEAROW1
-	dbw  1, MEOWTH5
+	dbw  2, SPEAROW_LV9
+	dbw  1, MEOWTH_LV17
 	dbw  1, ENERGY_RETRIEVAL
 	dbw  2, POTION
 	dbw  1, GUST_OF_WIND
 	dbw  2, SWITCH
 	dbw  1, BILL
 	db 0
-; 0x59637
+
+SweatAntiGR1Deck: ; 
```

Comment:
You can remove all of these empty ` ; ` comments.

Reply:
I added deck addresses here to my recent pr update